classes/PodsUI.php

```diff
@@ -3002,13 +3015,37 @@ public function manage( $reorder = false ) {
 									<label for="pods-form-ui-filter-<?php esc_attr_e( $filter ); ?>_start">
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
-									<?php echo PodsForm::field( 'filter_' . $filter . '_start', $start, $filter_field['type'], $filter_field ); ?>
+									<?php
+										// Prevent p div issues.
+										echo str_replace(
```

Comment:
:no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'str_replace'.

<sub>hash: b965cc58400e44da320ed22a36ce98d1</sub>